app/(pages)/content/[collection]/edit/_components/EditorPage.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react';
+import Link from 'next/link';
+import Image from 'next/image';
+import formConfig from '../../../../../../editor.config';
```

Comment:
Feel free to use the Next.js config shortcuts or add your own so you don't have to do this. Also if you just do '/' from the start it should start you from the root folder so rather than working backwards with '../' you can just start from the root.